src/bokeh/core/property/color.py

```diff
@@ -169,10 +170,10 @@ def transform(self, value: Any) -> Any:
 
 class Alpha(Percent):
 
-    _default_help = """\
+    _default_help = dedent("""\
     Acceptable values are floating-point numbers between 0 and 1 (0 being
     transparent and 1 being opaque).
-    """
+    """)
```

Comment:
TBH I think just dedenting the text inside the triple quoted string all the way would be fine, rather than actually calling `dedent`. It's just not a habit I'd want to get into in general (these things do contribute to import time), so I'd rather head it off from the start. 

Alternatively, I'd probably myself just define `ALPHA_DEFAULT_HELP` outside the class and then assign that to `default_help`

Reply:
I decided to go for the variable `ALPHA_DEFAULT_HELP` and `COLOR_DEFAULT_HELP` and get rid of some duplicated documentation, too. Hope it is fine now.